openhands/server/config/server_config.py

```diff
@@ -27,30 +27,32 @@ class ServerConfig(ServerConfigInterface):
         'openhands.server.user_auth.default_user_auth.DefaultUserAuth'
     )
 
-    def verify_config(self):
+    def verify_config(self) -> None:
         if self.config_cls:
             raise ValueError('Unexpected config path provided')
 
-    def get_config(self):
-        config = {
-            'APP_MODE': self.app_mode,
+    async def get_config(self) -> dict[str, str]:
```

Comment:
```suggestion
    def get_config(self) -> dict[str, str]:
```

This shouldn't be async